sklearn/model_selection/tests/test_search.py

```diff
@@ -1,20 +1,21 @@
 """Test the search module"""
 
+from collections import Iterable, Sized
```

Comment:
```py
from sklearn.utils.fixes import _Iterable as Iterable, _Sized as Sized
```
otherwise we get collections ABC warning in Python 3.7

Reply:
ah, right. not sure why @janvanrijn changed that.

Reply:
My code is an (old) version of master. I did not intentionally change that

Reply:
ah, that makes sense.